poolmgr/gp.go

```diff
@@ -262,6 +264,31 @@ func (gp *GenericPool) specializePod(pod *v1.Pod, metadata *fission.Metadata) er
 		return errors.New(fmt.Sprintf("Error from fetcher: %v", resp.Status))
 	}
 
+	logReq := logger.LogRequest{
```

Comment:
can you move this whole thing into a separate function?  So we can just have `gp.setupLogging(pod)` here